Developers/Unity Project/Spooksville/Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerMovement : MonoBehaviour
+{
+    public string[] maskedLayers;
```

Comment:
Better to just do something like this:
```suggestion
    public LayerMask maskedLayers;
```
Then you get a nice checkbox in the editor GUI and don't have to pass it through `LayerMask.GetMask` before using it.